official/enfuse.lua

```diff
@@ -149,6 +150,14 @@ dt.register_lib(
             cnt = cnt + 1
             f:write(i.path.."/"..i.filename.."\n")
             target_dir = i.path
+          elseif i.is_raw and darktable_cli_installed then
+            local tmp_exported = os.tmpname()..".tif"
+            os.execute("darktable-cli \""..i.path.."/"..i.filename.."\" \""..tmp_exported.."\"")
```

Comment:
@wpferguson does this construct with windows?

Reply:
@wpferguson that's definitely a better approach to mine. I used the same approach used for invoking `enfuse`, and I carefully checked if `darktable-cli` is in the path.

I will change it asap!